README.md

```diff
@@ -1,250 +1,163 @@
 # Minishift
 
-[![Build Status](https://secure.travis-ci.org/minishift/minishift.png)](https://travis-ci.org/minishift/minishift)
+Minishift is a tool that helps you to run OpenShift locally by running a single-node OpenShift
+cluster inside a VM. You can try out OpenShift or develop with it day-to-day on your local host.
+
+Minishift uses [libmachine](https://github.com/docker/machine/tree/master/libmachine) for
+provisioning VMs, and [OpenShift Origin](https://github.com/openshift/origin) for running the cluster.
+
+----
 
-## What is Minishift?
+<!-- MarkdownTOC -->
 
-Minishift is a tool that makes it easy to run OpenShift locally. Minishift runs a single-node OpenShift cluster inside a VM on your laptop for users looking to try out OpenShift or develop with it day-to-day.
+- [Installation](#installation)
+  - [Prerequisites](#prerequisites)
+  - [Instructions](#instructions)
+- [Quickstart](#quickstart)
+  - [Starting Minishift](#starting-minishift)
+  - [Reusing the Docker daemon](#reusing-the-docker-daemon)
+- [Documentation](#documentation)
+- [Known Issues](#known-issues)
+- [Community](#community)
 
-### Features
+<!-- /MarkdownTOC -->
+
+[![Build Status](https://secure.travis-ci.org/minishift/minishift.png)](https://travis-ci.org/minishift/minishift)
+[![Build status](https://ci.appveyor.com/api/projects/status/6wyv1cpd588cm4ce/branch/master?svg=true)](https://ci.appveyor.com/project/hferentschik/minishift-o61ou/branch/master)
 
-* Minishift packages and configures a Linux VM, Docker and all OpenShift components, optimized for local development
-* Minishift supports OpenShift features such as:
-  * DNS
-  * NodePorts
-  * ConfigMaps and Secrets
-  * Console
+----
 
+<a name="installation"></a>
 ## Installation
 
-### Requirements
+<a name="prerequisites"></a>
+### Prerequisites
+
+Minishift requires a hypervisor to run the VM containing OpenShift. Depending on your
+host OS, you have the following choices:
 
 * OS X
-    * [xhyve driver](DRIVERS.md#xhyve-driver), [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [VMware Fusion](https://www.vmware.com/products/fusion) installation
+    * [xhyve driver](./docs/docker-machine-drivers.md#xhyve-driver), [VirtualBox](https://www.virtualbox.org/wiki/Downloads) or [VMware Fusion](https://www.vmware.com/products/fusion) installation
```

Comment:
@praveenkumar +1. I would suggest we remove it as we have not tested it at all. Or atleast mention that it is not tested.